.github/workflows/build_and_deploy.yml

```diff
@@ -17,6 +17,7 @@ permissions:
 
 jobs:
   build:
+    if: contains(github.event.pull_request.labels.*.name, 'deploy') || github.event_name != 'pull_request'
```

Comment:
we don't normally have an if statement here, as we build on PR and merge. But the on pull_request usually has a "types: [opened, reopened, synchronize, labeled]"